### PR TITLE
Update to use python 3.6 since python 3.4 is deprecated

### PR DESCRIPTION
**Pull request by abs51295**

### CODE REVIEW

**Comment by riuvshin** [comment]
[test]

**Comment by abs51295** [comment]
[test]

**Comment by riuvshin** [comment]
[test]

File: base/fabric8-analytics-data-model/Dockerfile [hunk 1]
```diff
@@ -1,6 +0,0 @@
-FROM registry.access.redhat.com/rhel7
```
Comment (riuvshin) [inline base/fabric8-analytics-data-model/Dockerfile]: please also remove this symlink https://github.com/rhdt/EL-Dockerfiles/blob/master/build_order/0-fabric8-analytics-data-model

Comment (abs51295) [reply base/fabric8-analytics-data-model/Dockerfile]: Removed now

**Comment by riuvshin** [comment]
[test]

**Comment by riuvshin** [comment]
@abs51295 now it fails on `base/fabric8-analytics-server`
with:

```
Error: Package: mercator-1-32.el7.x86_64 (msrb-mercator)
           Requires: python34-toml
**********************************************************************
yum can be configured to try to resolve such errors by temporarily enabling
disabled repos and searching for missing dependencies.
To enable this functionality please set 'notify_only=0' in /etc/yum/pluginconf.d/search-disabled-repos.conf
**********************************************************************

 You could try using --skip-broken to work around the problem
[91mError: Package: mercator-1-32.el7.x86_64 (msrb-mercator)
           Requires: python34-toml
[0m You could try running: rpm -Va --nofiles --nodigest
The command '/bin/sh -c yum-config-manager --enable rhel-7-server-optional-rpms &&     yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm &&    yum install -y gcc patch git python36-pip python36-requests httpd httpd-devel                    python36-devel postgresql-devel redhat-rpm-config libxml2-devel                    libxslt-devel python36-pycurl mercator &&    yum clean all' returned a non-zero code: 1
```

**Comment by abs51295** [comment]
@msrb can you please have a look at this?

**Comment by riuvshin** [comment]
@abs51295 I think there is still 3.4 python here https://github.com/rhdt/EL-Dockerfiles/blob/master/base/fabric8-analytics-server/Dockerfile#L8

**Comment by abs51295** [comment]
@riuvshin I have already update it to python 3.6 in this PR

**Comment by riuvshin** [comment]
[test]

**Comment by abs51295** [comment]
@riuvshin it won't work because @msrb 's mercator RPM requires `python34`. We need to update that RPM first to accept `python36` as well.

**Comment by riuvshin** [comment]
@abs51295 ah right, sorry, so that fails on mercator since it depends on py 3.4, seems we need newer version of mercator that works with 3.6 right? 
cc @msrb

**Comment by abs51295** [comment]
Yep exactly @riuvshin :)

**Comment by abs51295** [comment]
@lucky-suman as discussed please use `pip3.4` and save the production for now :)

**Comment by aditya-konarde** [comment]
[test]

**Comment by riuvshin** [comment]
[test]

**Comment by riuvshin** [comment]
tests are passed!

**Pull request merged**